zeppelin-plugins/launcher/k8s-standard/src/test/java/org/apache/zeppelin/interpreter/launcher/K8sRemoteInterpreterProcessTest.java

```diff
@@ -123,7 +123,7 @@ void testGetTemplateBindings() {
     assertEquals("12321:12321" , p.get("zeppelin.k8s.interpreter.rpc.portRange"));
     assertEquals("zeppelin.server.service" , p.get("zeppelin.k8s.server.rpc.service"));
     assertEquals(12320 , p.get("zeppelin.k8s.server.rpc.portRange"));
-    assertEquals("null", p.get("zeppelin.k8s.interpreter.user"));
+    assertEquals("zeppelin", p.get("zeppelin.k8s.interpreter.user"));
```

Comment:
This value is not good. If no user is specified, the value should still be "null". It would be better if the property is not set for an empty user at all and the Jinja2 template does not render the user. Can you change the behaviour like this?

Reply:
I didn't programme cleanly when introducing the user in the template, sorry about that.